ui/plugin-lifecycle/rollup.config.js

```diff
@@ -0,0 +1,50 @@
+// rollup.config.js
+import angular from 'rollup-plugin-angular'
+import typescript from 'rollup-plugin-typescript'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import sass from 'node-sass'
+import tsc from 'typescript'
+
+export default {
```

Comment:
You've got a rollup config defined here and in your gulpfile.  If they are redundant, please reduce to a single config.

Reply:
rollup.config.js is removed in inital commit.